src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Compatibility/Extensions/IndentedStringBuilder.cs

```diff
@@ -39,54 +39,62 @@ public IndentedStringBuilder(StringBuilder stringBuilder)
 			_stringBuilder = stringBuilder;
 		}
 
-		public virtual IDisposable Indent(int count = 1)
+		public IDisposable Indent(int count = 1)
 		{
 			CurrentLevel += count;
 			return new DisposableAction(() => CurrentLevel -= count);
 		}
 
-		public virtual IDisposable Block(int count = 1)
+		public IDisposable Block(int count = 1)
 		{
 			var current = CurrentLevel;
 
 			CurrentLevel += count;
-			Append("{".Indent(current));
+			AppendIndented("{", current);
 			AppendLine();
 
 			return new DisposableAction(() =>
 			{
 				CurrentLevel -= count;
-				Append("}".Indent(current));
+				AppendIndented("}", current);
 				AppendLine();
 			});
 		}
 
-		public virtual IDisposable Block(IFormatProvider formatProvider, string pattern, params object[] parameters)
+		public IDisposable Block(IFormatProvider formatProvider, string pattern, params object[] parameters)
 		{
-			AppendFormat(formatProvider, pattern, parameters);
+			AppendFormatIndented(formatProvider, pattern, parameters);
 			AppendLine();
 
 			return Block();
 		}
 
-		public virtual void Append(string text)
+		public void Append(string text)
 		{
 			_stringBuilder.Append(text);
 		}
 
-		public virtual void AppendFormat(IFormatProvider formatProvider, string pattern, params object[] replacements)
+		public void AppendIndented(string text)
 		{
-			_stringBuilder.AppendFormat(formatProvider, pattern.Indent(CurrentLevel), replacements);
+			AppendIndented(text, CurrentLevel);
+		}
+
+		private void AppendIndented(string text, int indentCount)
+		{
+			_stringBuilder.Append(new string('\t', indentCount)); // TODO: Which is faster? this? or looping and appending a single tab each iteration?
```

Comment:
We should use this : https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.append?view=net-6.0#system-text-stringbuilder-append(system-char-system-int32)